### ISSUE TITLE
fails to run

### ISSUE THREAD

**Issue by jonsowman** [BeerNCode/punchykickgravityflipwarz#9]
```
pygame 1.9.6
Hello from the pygame community. https://www.pygame.org/contribute.html
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/jon/projects/punchykickgravityflipwarz/punchykickgravityflipwarz/__main__.py", line 3, in <module>
    from punchykickgravityflipwarz.game import Game
  File "/home/jon/projects/punchykickgravityflipwarz/punchykickgravityflipwarz/game.py", line 4, in <module>
    from punchykickgravityflipwarz.item import *
  File "/home/jon/projects/punchykickgravityflipwarz/punchykickgravityflipwarz/item.py", line 136
    <<<<<<< HEAD
    ^
IndentationError: expected an indented block
```

**Comment by jonsowman** [comment]
can someone please fix this please I need it uregently

**Comment by jonsowman** [comment]
fixed in 739c9f7f4a833cfd575bf937059ea3c0b88cf3a1
